src/client/fuse_ll.cc

```diff
@@ -1560,7 +1566,8 @@ uint64_t CephFuse::Handle::make_fake_ino(inodeno_t ino, snapid_t snapid)
     int first = last_stag & STAG_MASK;
     int stag =  (++last_stag) & STAG_MASK;
     for (; stag != first; stag = (++last_stag) & STAG_MASK) {
-      if (stag == 0)
+      // two reserved stags: 0 for CPEH_NOSNAP and 1 for CEPH_SNAPDIR
+      if (stag == 0 || stag == 1)
```

Comment:
nit: typo correction
`s/CPEH_NOSNAP/CEPH_NOSNAP/`

Reply:
Sure, will fix it.